src/main/java/fi/okm/jod/yksilo/controller/profiili/TyopaikkaController.java

```diff
@@ -46,12 +47,12 @@ List<TyopaikkaDto> getAll(@AuthenticationPrincipal JodUser user) {
 
   @PostMapping
   @ResponseStatus(HttpStatus.CREATED)
-  ResponseEntity<Void> add(
+  ResponseEntity<IdDto<UUID>> add(
       @Validated(Add.class) @RequestBody TyopaikkaDto dto, @AuthenticationPrincipal JodUser user) {
     var id = service.add(user, dto);
     var location =
         ServletUriComponentsBuilder.fromCurrentRequest().path("/{id}").buildAndExpand(id).toUri();
-    return ResponseEntity.created(location).build();
+    return ResponseEntity.created(location).body(new IdDto<>(id));
```

Comment:
Technical specification delta: return body IdDto instead of list of objects

Reply:
won't fix, spec will be changed